llama2.mojo

```diff
@@ -272,6 +273,18 @@ struct FileBuf:
         return self.offset
 
 
+fn wrap(token: PointerString) -> PointerString:
```

Comment:
Potentially this transformation could be moved to vocabulary pre-loading part. WDYT?

Reply:
Pre-loading method doesn't work, becasue our tokenizer will **only** identify <0x0A> as '\n'.

However, if I substitute '\n' with '<0x0A>', the tokenizer can't tokenize this correctly.

Example of preloading
prompt = 'hi\nI like'
processed_pronpt = 'hi<0x0A>I like'
 => [hi, <, 0, x0, A,>I, like]

Reply:
> Potentially this transformation could be moved to vocabulary pre-loading part. WDYT?

I think we can do transformation more early. May be we can do this in the tokenizer.py which change a tokenizer.model to tokenizer.bin.